component/prometheus/remotewrite/cli.go

```diff
@@ -0,0 +1,210 @@
+package remotewrite
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+
+	"github.com/grafana/agent/pkg/agentctl"
+	"github.com/olekukonko/tablewriter"
+	"github.com/spf13/cobra"
+)
+
+// InstallTools installs command line utilities as subcommands of the provided
+// cmd.
+func InstallTools(cmd *cobra.Command) {
+	cmd.AddCommand(
+		samplesCmd(),
+		targetStatsCmd(),
+		walStatsCmd(),
+	)
+}
+
+func samplesCmd() *cobra.Command {
```

Comment:
These have been copied over nearly verbatim (minus removing references to `agentctl`) from agentctl.